src/mp_integer.hpp

```diff
@@ -2294,6 +2294,21 @@ class mp_integer
 		{
 			return mpz_view(*this);
 		}
+		/// Get an \p non-const mpz_ptr to dynamic storage of \p this.
```

Comment:
I imagine here it should be ``non-const \p mpz_ptr`` rather than ``\p non-const mpz_ptr``? Same in the ``@return`` directive.